repos/fdbt-site/tests/pages/products/__snapshots__/selectExports.test.tsx.snap

```diff
@@ -642,6 +642,11 @@ exports[`selectExports renders fully when the user has products they can export,
                 className="govuk-tabs__panel"
                 id="services"
               >
+                <h2
+                  className="govuk-heading-m govuk-!-margin-bottom-6"
+                >
```

Comment:
can this be one line?

Reply:
This file is autogenerated by the snapshot test, its not code we create.